packages/discovery-react-components/src/components/DocumentPreview/__tests__/DocumentPreview.test.tsx

```diff
@@ -5,18 +5,64 @@ import { NoAuthAuthenticator } from 'ibm-watson/auth';
 import DiscoveryV2 from 'ibm-watson/discovery/v2';
 import DiscoverySearch from 'components/DiscoverySearch/DiscoverySearch';
 import DocumentPreview from '../DocumentPreview';
-import { document as doc } from '../__fixtures__/Art Effects.pdf';
-import docJson from '../__fixtures__/Art Effects Koya Creative Base TSA 2008.pdf.json';
+import pdfDoc from '../__fixtures__/Art Effects Koya Creative Base TSA 2008.pdf.json';
 import passages from '../__fixtures__/passages';
+import 'utils/test/createRange.mock';
+import { SearchApiIFC, SearchContextIFC } from 'components/DiscoverySearch/DiscoverySearch';
+import { wrapWithContext } from 'utils/testingUtils';
 
 describe('DocumentPreview', () => {
+  let getByText: NonNullable<any>, findAllByTestId: NonNullable<any>, findByText: NonNullable<any>;
+
+  const api: Partial<SearchApiIFC> = {};
+  let context: Partial<SearchContextIFC> = {};
+
   const mockedBbox = {
     x: 0,
     y: 0,
     width: 10,
     height: 12
   };
 
+  const htmlTextDoc = {
+    id: '1234567890',
+    extracted_metadata: {
+      filename: 'i_am_a_pdf_file.pdf',
+      file_type: 'pdf'
+    },
+    text: 'Example text <text> <username> <password> more text afterwards.',
+    body_field: 'Display text from the specified "body" field.'
+  };
+
+  const jsonDoc = {
+    id: '1234567890',
+    extracted_metadata: {
+      filename: 'i_am_a_json_file.json',
+      file_type: 'json'
+    },
+    html: '<p>This field should not get picked up.</p>',
```

Comment:
This statement is more of a test statement, not a data statement. Instead, just use something like `<p>This is an <em>HTML</em> string</p>`.